Entities/Industry/Drill/Drill.as

```diff
@@ -176,20 +177,20 @@ void onTick(CBlob@ this)
 	sprite.SetEmitSoundPaused(true);
 	if (this.isAttached())
 	{
-		this.getCurrentScript().runFlags &= ~(Script::tick_not_sleeping);
```

Comment:
I don't think this was supposed to be removed.
I noticed there's a lot of bugs with the drill after this PR.

Other classes can use the drill (no rotation).
The drill doesn't cool while held in water.
Probably others.